sources/ElkArte/Unread.php

```diff
@@ -633,6 +506,6 @@ private function _getUnreadReplies($start, $limit, $include_avatars = false)
 		}
 		$request->free_result();
 
-		return TopicUtil::prepareContext($return, true, ((int) $this->_preview_bodies) + 128);
+		return Topic_Util::prepareContext($return, true, ((int) $this->_preview_bodies) + 128);
```

Comment:
This seems to be not correct. I think the class name was changed to TopicUtil in 2.0

Reply:
Oops you are correct!  I caught some of those during the munge ... thats what made this so much fun LOL

Reply:
I can imagine! :rofl: